src/App.tsx

```diff
@@ -1,156 +1,85 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Header } from './Components/Header';
+import { Main } from './Components/Main';
+import { Footer } from './Components/Footer';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+
+const USER_ID = 10307;
+
+type Errors = {
+  loading?: boolean;
+  posting?: boolean;
+  editing?: boolean;
+  deleting?: boolean;
+};
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  // if (!USER_ID) {
+  //   return <UserWarning />;
+  // }
+
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [errors, setErrors] = useState<Errors | null>(null);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
```

Comment:
TS already knows about it, bc you put an initial value
```suggestion
  const [filter, setFilter] = useState(Filter.All);
```